hw2/03-count.html

```diff
@@ -6,28 +6,42 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
+  <body class="bg-dark">
```

Comment:
Expected background color is some shade of blue, but even I am having difficulty trying to figure out the color code.

Reply:
I was confused on how we were supposed to implement Bootstrap styling with a color that wasn't in any of the Bootstrap options so I went with what I thought looked best.